installer/validate/authenticate.py

```diff
@@ -0,0 +1,111 @@
+#
```

Comment:
why there are 2 similar files - `/tests/management_api_tests/authenticate.py` ?

Reply:
I'm gonna remove that, we don't use it anymore